prow/cmd/pipeline/controller.go

```diff
@@ -339,7 +339,7 @@ func (c *controller) createPipelineResource(context, namespace string, pr *pipel
 	if err != nil {
 		return nil, err
 	}
-	return pc.client.TektonV1alpha1().PipelineResources(namespace).Create(pr)
+	return pc.resourceClient.TektonV1alpha1().PipelineResources(namespace).Create(pr)
```

Comment:
Also not really related to the pull itself, but shouldn't we update the pipelineresource after the pipelinerun was created with an owner ref[0] pointing to the pipelinerun, so that they get deleted when the pipelinerun gets deleted?

[0] https://kubernetes.io/docs/concepts/workloads/controllers/garbage-collection/#owners-and-dependents

Reply:
I don't think we need to watch PipelineResources as we don't react to PipelineResource changes.

Reply:
We definitely should update PipelineResource owner but i will do that in another PR.
In order, we should create the PipelineResource, then create the PipelineRun, and finally update the PipelineResource to set the owner.